RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -261,7 +261,7 @@ class ObjectCreationTests: TestCase {
         XCTAssertEqual(stringObjects.count, 1)
         let persistedObject = object.object!
 
-        XCTAssertEqual(persistedObject.intCol, 12)
+        XCTAssertEqual(persistedObject.intCol, 11)
```

Comment:
because it's also the new value -- `standalone` is also passed into the `create` call